cmp/grid/renderers/MultiFieldRenderer.ts

```diff
@@ -30,25 +30,37 @@ export function multiFieldRenderer(value, context): ReactNode {
     );
 
     const {mainRenderer, delimiter, subFields = []} = multiFieldConfig,
-        [topFields, bottomFields] = partition(subFields, it => it.position === 'top'),
-        topRowItems = [],
+        [topFields, bottomFields] = partition(subFields, it => it.position === 'top');
+
+    let topRowItems = [],
         bottomRowItems = [];
 
     // Render main field to top row
     topRowItems.push(renderMainField(value, mainRenderer, context));
 
     // Render SubFields to top row
-    topFields.forEach(it => {
-        if (delimiter) topRowItems.push(renderDelimiter(delimiter));
+    topFields.forEach((it, idx) => {
```

Comment:
use map here

Reply:
can't. please see the special handling on line 39 to push the "main field" to the same array